CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ It is possible to override this in `.ic-assets.json` like this:
 }
 ```
 
+### fix: Transferring funds too early in `dfx ledger create-canister` with --next-to
+
+When creating a canister with `dfx ledger create-canister --next-to` on a canister that does not exist (e.g., aaaaa-aa), then the funds are first transferred away from the users account, but the call then fails to create the new canister, and the funds are not returned to the user's account.
```

Comment:
aaaaa-aa exists (it's the management canister!), but it isn't a canister deployed to a specific subnet; let's pick a different canister ID for the example

Reply:
But thats a canister ID with which it _always_ fails. All other canister IDs could exist. What should I use as an example?

Reply:
> All other canister IDs could exist

a self-authenticating principal (user identity), for instance; or this canister ID `37pka-5h777-77777-7777q-cai` which is derived from u64::MAX (unlikely to ever be used on the IC mainnet)

Reply:
How about `Principal::anonymous?`